media/css/base/send-to-device.less

```diff
@@ -0,0 +1,594 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@import "../sandstone/lib.less";
+
+#send-to-device {
+    width: 100%;
+    margin: 0 auto;
+    background: #f2f2f2;
+    color: #56565a;
+    text-align: center;
+
+    .form-container {
+        position: relative;
+        width: @widthDesktop - (@gridGutterWidth * 2);
+        padding-top: (@baseLine * 2);
+    }
+
+    h2 {
+        margin: 0 auto;
+        padding: 0 @baseLine (@baseLine * 2);
+        color: #56565a;
+        .font-size(42px);
+        text-shadow: none;
+    }
+
+    ul {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+
+        li {
+            margin: 0;
+            padding: 0;
+        }
+    }
+
+    footer {
+        margin-top: @baseLine;
+        padding: (@baseLine / 2) 0;
+        background: #e1e2e2;
+        border-top: 1px solid #d3d4d4;
+        .clearfix();
+
+        ul {
+            position: relative;
+
+            li {
+                float: left;
+                width: 50%;
+
+                a {
+                    padding: (@baseLine / 2) (@baseLine * 2);
+                    .font-size(@largeFontSize);
+                    line-height: 4;
+                }
+
+                &.app-store {
+                    text-align: right;
+                }
+
+                &.google-play {
+                    text-align: left;
+                }
+
+                &:nth-child(2) a {
+                    border-left: 1px dotted #56565a;
+                }
+            }
+
+            &.ios .google-play {
+                display: none;
+            }
+
+            &.android .app-store {
+                display: none;
+            }
+
+            &.ios,
+            &.android {
+                li {
+                    float: none;
+                    width: 100%;
+                    text-align: center;
+
+                    &:nth-child(2) a {
+                        border-left: none;
+                    }
+                }
+            }
+        }
+    }
+
+    a {
+        color: #0c99d5;
+        .transition(color .1s ease-in-out);
+
+        &:active,
+        &:hover,
+        &:focus {
+            color: darken(#0c99d5, 5%);
```

Comment:
Hmm, I just guessed at this 5% darker value. I'm going to hold off adding the computed color value to lib.less for now, as it may not correctly match what's used elsewhere.